pydantic/main.py

```diff
@@ -19,14 +20,26 @@
 from .validators import dict_validator
 
 
+class ExtraAttributes(Enum):
+    DISALLOW_MUTATION = auto()
```

Comment:
I don't think we should have different behaviour for mutation and init, Just `ignore` and `allow` should suffice.

Reply:
also, please use lower case.

Reply:
sorry, `ignore`, `allow` and `disallow`.

But since we're making a change, maybe `ignored`, `allowed` and `forbidden` would be clearer.